mypy/nodes.py

```diff
@@ -155,9 +155,6 @@ def accept(self, visitor: StatementVisitor[T]) -> T:
 
 class Expression(Node):
     """An expression node."""
-    literal = LITERAL_NO
-    literal_hash = None  # type: Key
```

Comment:
And yes, if this is not a declaration, then you should use explicitly ``Optional[Key]`` here.